### DISCUSSION TITLE
Corrupt files with start on speed

### DISCUSSION THREAD

**Discussion by TripleFJeff** [gopro/labs#938] [Q&A]
I have a Hero 8 setup to wake on power and start on speed. Up until now I have had no issues with corrupt files. I moved one, with same card, to a different location. The only difference is that the power supply is taken from a place that is on a "power saver" circuit. The previous location had power whenever the car master switch is on. This location may shut off after sitting with ignition off. But with ignition on it always has power. I've gotten several corrupted files in this location. Any ideas how to troubleshoot?

Also, any way to recover the files? The camera doesn't seem to fix them on startup.

**Comment by dnewman-gpsw** [comment]
What is a the command?  Share an example?

**Reply by TripleFJeff** [reply]
This is my full setup after a reset:

g1!MBRNO=10!MBURN="(0,40)[BCHH:MM:SS AA mm-dd-yyyy ]" (burnin)

!MWAKE=2 (wake on power)

!MBOOT="!Lspd"!SAVEspd=!SK32H30!R (start on speed boot command)

M64BT=1 (12GB Files)

mVr1080p60fSt0e1hS0aS (Basic settings 1)

g1q0dVoV0B4D4R0C0S1 (Basic settings 2)

**Reply by dnewman-gpsw** [reply]
Can you share an corrupt file?

**Reply by TripleFJeff** [reply]
Emailed you a link to a file

**Reply by dnewman-gpsw** [reply]
The file is truncated, and not likely due to any of the above.  Set !M64BT=0 to see this problem goes away.

**Reply by TripleFJeff** [reply]
By truncated, you mean that it was stopped in the middle of recording as if it lost power? This same camera and card has worked flawlessly in a different location, but the power circuit in this location is different. May need to fix that.

That file is 1.5GB, so why would changing !M64BT make a difference? (just trying to understand how it works)

Any way to recover?

**Reply by TripleFJeff** [reply]
Update-- going back to the card, it seems that the camera did repair the file. But the repaired file GH010270 is 5.7GB, while the one I sent you of the same filename was only 1.45GB. I had restarted it with the card in previously and it hadn't repaired it.

**Reply by dnewman-gpsw** [reply]
So yes, like I said it was truncated. The header said to the file should have been greater than 4GB.  This is not a lost power error. Are you using the USB port to transfer media from the camera?  Get this media using the SD card, that is faster and should never truncate a file.

**Reply by TripleFJeff** [reply]
Did trying to download via USB truncate the video on the card, or just the file transferred over USB? I did download via USB and that is when I first had the problem. Then I took the card out, and am pretty sure that it was also corrupt on the card. I then put the card back in the camera and used it the next day, and after that the original file was OK. But I'm not 100% sure that is how it happened.

The camera is in a place that is difficult to reach to get the card, but I can easily get to the end of the USB cable that powers it, that is why I downloaded via USB.
